src/core/net/srp_client.hpp

```diff
@@ -1076,6 +1083,7 @@ class Client : public InstanceLocator, private NonCopyable
     uint16_t mUpdateMessageId;
     uint16_t mAutoHostAddressCount;
     uint32_t mRetryWaitInterval;
+    bool     mShouldUseLongDelays;
```

Comment:
Rename to `mUseUpdateTxDelayLong`? To mirror the `kUpdateTxMaxDelay` name?

Reply:
Sure, agree. Modified this